cmd/version.go

```diff
@@ -28,11 +28,13 @@ const (
 // buildVersion is a display format of the version and build metadata in compliance with semver.
 func buildVersion() string {
 	// noinspection GoBoolExpressions
-	if SCMCommit == "" || strings.Contains(Version, SCMCommit) {
+
+	sha := strings.TrimSuffix(SCMCommit, "-dirty")
```

Comment:
Maybe we should just use the same commit in both the version output and the lifecycle tag, either with or without dirty seems fine. Do we have a reason to prefer/maintain this difference?

Reply:
should we just use the version without `-dirty` in both places then?

Reply:
I think it's a good idea to have consistency here.
Maybe we can remove `-dirty` for now and open a new issue for adding it in all places.

Reply:
I am fine with this. `-dirty` provides a little extra information for local development but it seems not worth the extra confusion in having to keep track of all these versions!